packages/zk-eddsa-ticket-pcd/artifacts-unsafe/generate_artifacts.sh

```diff
@@ -0,0 +1,30 @@
+rm -rf eddsaTicket_js
+rm eddsaTicket_0000.zkey
+rm eddsaTicket_0001.zkey
+rm eddsaTicket.r1cs
+rm eddsaTicket.zkey
+rm verification_key.json
+rm -rf ../../../apps/passport-client/public/zkeddsa-artifacts-unsafe
+rm -rf ../../../apps/passport-server/public/zkeddsa-artifacts-unsafe
+rm -rf ../../../apps/consumer-client/public/zkeddsa-artifacts-unsafe
+
+circom eddsaTicket.circom --r1cs --wasm
+snarkjs groth16 setup eddsaTicket.r1cs powersOfTau28_hez_final_13.ptau eddsaTicket_0000.zkey
+snarkjs zkey contribute eddsaTicket_0000.zkey eddsaTicket_0001.zkey --name="Unsafe devmode contribution" -v
+snarkjs zkey beacon eddsaTicket_0001.zkey eddsaTicket.zkey 0102030405060708090a0b0c0d0e0f101112131415161718191a1b1c1d1e1f 10 -n="Final Beacon phase2"
+snarkjs zkey export verificationkey eddsaTicket.zkey verification_key.json
+
+mkdir ../../../apps/passport-client/public/zkeddsa-artifacts-unsafe
+cp eddsaTicket.zkey ../../../apps/passport-client/public/zkeddsa-artifacts-unsafe
+cp verification_key.json ../../../apps/passport-client/public/zkeddsa-artifacts-unsafe
+cp eddsaTicket_js/eddsaTicket.wasm ../../../apps/passport-client/public/zkeddsa-artifacts-unsafe
+
+mkdir ../../../apps/passport-server/public/zkeddsa-artifacts-unsafe
+cp eddsaTicket.zkey ../../../apps/passport-server/public/zkeddsa-artifacts-unsafe
+cp verification_key.json ../../../apps/passport-server/public/zkeddsa-artifacts-unsafe
+cp eddsaTicket_js/eddsaTicket.wasm ../../../apps/passport-server/public/zkeddsa-artifacts-unsafe
+
+mkdir ../../../apps/consumer-client/public/zkeddsa-artifacts-unsafe
+cp eddsaTicket.zkey ../../../apps/consumer-client/public/zkeddsa-artifacts-unsafe
+cp verification_key.json ../../../apps/consumer-client/public/zkeddsa-artifacts-unsafe
+cp eddsaTicket_js/eddsaTicket.wasm ../../../apps/consumer-client/public/zkeddsa-artifacts-unsafe
```

Comment:
i think there is probably some better way to engineer this script such that it doesn't depend on being at a particular spot in the monorepo (specifically regarding the various `../`), eg by putting the artifacts in a well-known spot in this pcd's package, and then copying from that place in its dependencies. if you don't want to do that here, we could create an issue with it. I also wonder what @cedoor thinks about this.

Reply:
What do you think about putting adding something to the `build` command of apps like consumer-client, something like `cp node_modules/@pcd/zk-eddsa-ticket-pcd/artifacts ./public`?

(By default I'd prefer to just create an issue for now and not address in this PR)

Reply:
opened an issue here and won't deal with this in this PR: https://github.com/proofcarryingdata/zupass/issues/466